### DISCUSSION TITLE
Roadmap To Support NVIDIA

### DISCUSSION THREAD

**Discussion by fakebizprez** [hyprwm/Hyprland#7877] [General]
Is there a target date in mind to officially support NVIDIA GPUs?

I won't use Linux as my daily driver without Wayland and a tiling manager, and the only tiling managers that support Wayland (Sway & Hyprland) don't support NVIDIA.

**Comment by lucifertdark** [comment]
I installed Hyprland on Fedora with an Nvidia RTX 3060ti & the 560 drivers & it's running just fine without official support. Tiling, Animation, Wallpapers, Plugins, the whole shebang. :)

**Reply by fakebizprez** [reply]
That's encouraging! I have two 4090 GPUs in this machine. 

Did this require any config modifications?

**Comment by vaxerski** [comment]
> officially

No.

Unofficially they are supported quite well I'd say, many nvidia users can use hyprland just fine.

**Reply by fakebizprez** [reply]
Yes, I've read this, and attempted to replicate the same results using Arch, by following the documentation (& other forum posts), but did not have success. 

I have real work to get done, I can't spend hours configuring an operating system/tiling manager to work with the most popular graphics card manufacturer on the planet. Which is why I'm asking when it will be officially supported, so the installation will be as easy as setting up any tiling manager of the past.

**Reply by vaxerski** [reply]
No, and it's not up to us. The reason you have to jump through the hoops is because of nvidia, not us.

**Reply by fakebizprez** [reply]
What obstacles are NVIDIA encountering, @vaxerski? 
I was under the impression that since releasing open-source drivers, the experience has been more pleasant for Linux developers.

I have been running the tiling manager when I use my laptop during travel. Great work.

**Reply by vaxerski** [reply]
nvidia has not released open source drivers. They are still closed source. Kernel modules are not the drivers.
